### DISCUSSION TITLE
Thanks! And an idea.

### DISCUSSION THREAD

**Discussion by robs1234** [roleoroleo/yi-hack-Allwinner#281] [Ideas]
@ roleoroleo,

Just wanted to say a big thank you for all the work and efforts you put into this - it's appreciated by many.

Can see you already have your hands full with the RTSP fix - looking forward to it!

Is there a way a timer could be added - to turn on/off the camera at specific time of day, or perhaps more usefully and more complex, at different times on different days of the week?

D

**Comment by roleoroleo** [comment]
You can use cron.

**Reply by robs1234** [reply]
Sweet! Couldn't get it working last night though - is the command -t ON and -t OFF ? Tried all sorts...
